pyaedt/desktop.py

```diff
@@ -677,99 +911,140 @@ def __getitem__(self, project_design_name):
-                else:
-                    warnings.warn(
-                        """PyAEDT has limited capabilities when used with an AEDT version earlier than 2021 R1.
-                        PyAEDT officially supports AEDT version 2021 R1 and later."""
-                    )
-            if student_version:
-                specified_version += "SV"
-                student_version_flag = True
-            assert specified_version in self.version_keys, "Specified version {} is not known".format(specified_version)
-            version_key = specified_version
-        else:
-            if student_version and self.current_version_student:
-                version_key = self.current_version_student
-                student_version_flag = True
+
+    def _assert_version(self, specified_version, student_version):
+        # avoid evaluating the env variables multiple times
+        self_current_version = self.current_version
+        self_current_student_version = self.current_student_version
+
+        if current_version == "":
+            raise Exception("AEDT is not installed on your system. Please install AEDT version 2022 R2 or higher.")
+        if not specified_version:
+            if student_version and self_current_student_version:
+                specified_version = self_current_student_version
+            elif student_version and self_current_version:
+                specified_version = self_current_version
+                student_version = False
+                self.logger.warning("AEDT Student Version not found on the system. Using regular version.")
             else:
-                version_key = self.current_version
-                student_version_flag = False
-        if student_version and student_version_flag:
-            version = "Ansoft.ElectronicsDesktop." + version_key[:-2]
-        else:
-            version = "Ansoft.ElectronicsDesktop." + version_key
-        self._main.sDesktopinstallDirectory = os.getenv(self._version_ids[version_key])
-        return student_version_flag, version_key, version
+                specified_version = self_current_version
+                if "SV" in specified_version:
+                    # specified_version = specified_version[0:6]
+                    student_version = True
+                    self.logger.warning("Only AEDT Student Version found on the system. Using Student Version.")
+        elif student_version:
+            specified_version += "SV"
+
+        if float(specified_version[0:6]) < 2019:
+            raise ValueError(
+                "PyAEDT supports AEDT version 2021 R1 and later. Recommended version is 2022 R2 or higher."
+            )
+        elif float(specified_version[0:6]) < 2022.2:
+            warnings.warn(
+                """PyAEDT has limited capabilities when used with an AEDT version earlier than 2022 R2.
+                Please update your AEDT installation to 2022 R2 or higher."""
```

Comment:
Technical documentation should avoid "please". Could this message string be updated to remove it? I assume that also means updating a test.